src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const copyState = {};
+
+  for (const key in state) {
+    copyState[key] = state[key];
+  }
```

Comment:
If you want to copy the whole object it's better to use `rest` operator or `Object.assign`